src/content/docs/en/guides/deploy/google-firebase.mdx

```diff
@@ -5,7 +5,9 @@ type: deploy
 i18nReady: true
 ---
 
-[Firebase Hosting](https://firebase.google.com/products/hosting) is a service provided by Google’s [Firebase](https://firebase.google.com/) app development platform, which can be used to deploy an Astro site.
+[Firebase Hosting](https://firebase.google.com/products/hosting) is a service provided by Google’s [Firebase](https://firebase.google.com/) app development platform, which can be used to deploy an Astro site. 
+
+See our separate guide for [adding a Firebase backend services](/en/guides/deploy/google-firebase/) such as databases, authentication, and storage.
```

Comment:
No indefinite plurals:

```suggestion
See our separate guide for [adding Firebase backend services](/en/guides/deploy/google-firebase/) such as databases, authentication, and storage.
```

Otherwise LGTM!